bin/editorconfig-el

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+:;#-*-Emacs-Lisp-*-
```

Comment:
Just one comment, can you add the license header to this file as well? This file seems significantly long enough, so without a license header its license would be ambitious---since we have a license in the root directory but no license (proprietary by default) in the header.